pages/content/amp-dev/documentation/guides-and-tutorials/contribute/integrate-your-analytics-tools.md

```diff
@@ -10,7 +10,7 @@ toc: true
 ---
 
 <!--
-This file is imported from https://github.com/ampproject/amphtml/blob/main/extensions/amp-analytics/integrating-analytics.md.
+This file is imported from https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/integrating-analytics.md.
```

Comment:
This looks like an outdated file -> should be main

Reply:
I will merge the latest changes again. Something went wrong there, I guess.

Reply:
This looks better now.